R/pkg/R/mllib_regression.R

```diff
@@ -84,6 +84,12 @@ setClass("IsotonicRegressionModel", representation(jobj = "jobj"))
 #' # can also read back the saved model and print
 #' savedModel <- read.ml(path)
 #' summary(savedModel)
+#'
```

Comment:
please update L56 for documentation. Also we should update the programming guide and vignettes too